server/src/main/scala/drt/server/feeds/common/XlsExtractorUtil.scala

```diff
@@ -21,4 +21,10 @@ object XlsExtractorUtil {
 
   val dateCell: (Int, Row) => Date = (index, row) => row.getCell(index).getDateCellValue
 
+  val headingIndexByName: Row => Map[Option[String], Int] = headingRow => (headingRow.getFirstCellNum to headingRow.getLastCellNum map { index =>
+    if (headingRow.getCell(index) != null && headingRow.getCell(1).getCellType != Cell.CELL_TYPE_BLANK) {
+      stringCell(index, headingRow) -> index
+    } else None -> index
+  }).toMap
```

Comment:
is it worth adding 
```
.collect {
  case (Some(s), index) => s -> index
}
```

to get rid of the options here?

Reply:
👍